dotcom-rendering/src/components/LatestLinks.importable.stories.tsx

```diff
@@ -158,26 +155,34 @@ export const WorldCupFinal2023 = () => {
 				background-color: ${overrides.background.container};
 			`}
 		>
-			<Island priority="critical">
-				<LatestLinks
-					id="/football/live/2023/aug/20/spain-v-england-womens-world-cup-final-live"
-					format={format}
-					direction="horizontal"
-					containerPalette={containerPalette}
-					isDynamo={true}
-				/>
-			</Island>
+			<ContainerOverrides
+				containerPalette={containerPalette}
+				isDynamo={true}
+			>
+				<Island priority="critical">
+					<LatestLinks
+						id="/football/live/2023/aug/20/spain-v-england-womens-world-cup-final-live"
+						direction="horizontal"
+						containerPalette={containerPalette}
+						isDynamo={true}
+					/>
+				</Island>
+			</ContainerOverrides>
 		</Wrapper>
 	);
 };
+WorldCupFinal2023.storyName = 'World Cup Final 2023';
+WorldCupFinal2023.decorators = [
+	lightDecorator([
```

Comment:
```suggestion
	splitThemeDecorator([
```

so that we can see dark mode as well?